libvirt/resource_libvirt_domain.go

```diff
@@ -126,7 +126,12 @@ func resourceLibvirtDomain() *schema.Resource {
 			"graphics": &schema.Schema{
 				Type:     schema.TypeMap,
 				Optional: true,
-				Required: false,
+				Computed: true,
```

Comment:
If you remove this line, the test should succeed. At least they do on my local machine.